### ISSUE TITLE
Code release

### ISSUE THREAD

**Issue by thangld201** [sufenlp/StyleBART#1]
Thanks for the wonderful work !

Could you please release the source codes ? @Eurus-W

**Comment by zhenghuawang6** [comment]
Thanks for the wonderful work !

Could you please release the source codes ? @Eurus-W

**Comment by Eurus-W** [comment]
> Thanks for the wonderful work !
> 
> Could you please release the source codes ? @Eurus-W

Sorry for the late reply, and thank you for your interest in our work. We have already released our core structure code, and the complete pipeline will be open-sourced soon.

**Comment by Eurus-W** [comment]
> Thanks for the wonderful work !
> 
> Could you please release the source codes ? @Eurus-W

We appreciate your interest in our work. We have made our core structure code available, and the complete pipeline will be open-sourced soon.

**Comment by zhenghuawang6** [comment]
very thanks for your responses, your code provided now is very concise and reasonable. However I am very curious about the calculation of metric PPL-S. I am looking forward to your updates soon. thanks again！！！

**Comment by Eurus-W** [comment]
> very thanks for your responses, your code provided now is very concise and reasonable. However I am very curious about the calculation of metric PPL-S. I am looking forward to your updates soon. thanks again！！！

Thank you for your kind words. We appreciate your curiosity about the calculation of the PPL-S metric. We are actively working on finalizing the details and will release the related code soon. Thank you for your patience!

Here is an introduction to the details of the PPL-S calculation, which we hope will be helpful to you:

We first fine-tune the pretrained GPT-2 model on different style-specific corpora, creating distinct style-GPT-2 models. Next, we calculate the perplexity (PPL) of style-specific headlines using the corresponding style-GPT-2 models. For example, we evaluate the perplexity of humor headlines with the humor-GPT-2 model, which was trained on a humor corpus.
